services/migration_service/requirements.txt

```diff
@@ -1,4 +1,3 @@
-aiohttp
-aiohttp_swagger
-psycopg2
-aiopg
\ No newline at end of file
+aiohttp==3.6.2
```

Comment:
is there a particular reason you pinned these dependencies?

Reply:
version pinning was introduced to dependencies as a result of a particular version of aiohttp depending on a version of yarl that broke url query parameter parsing. it is debatable if the pinning is necessary for all services, as for now, only the ui backend relies heavily on parsing params.

Reply:
pinning dependencies in general should be avoided because it will probably cause issues down the line. Ideally we can resolve the issue with url parsing, otherwise we can pin only the dependencies we need to pin